### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 24

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#296] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152320701-f46fe5d1-a9fe-4cf9-bfa0-0885ab70cf7c.png)

**Comment by LuginiAndrea** [comment]
La transitività non è implicata: basti pensare a un caso dove $\forall a \in A, \exists! b \in A \textrm{ t.c. } aRb$.
In altre parole, $aRb, bRc, \nexists aRc$

La simmetria e la transitività implicano la riflessività, unito al fatto che $\forall a \in A \exists b \in A aRb$: $aRb \wedge bRa \Longrightarrow aRa$

Ciò non vale al contrario per la transitività: $aRb, bRa, bRc, cRb, aRa, bRb, cRc \Longrightarrow \nexists aRc$

**Comment by UnoCasuale** [comment]
Non sono d'accordo sulla seconda in quanto se A = {1}, A può essere simmetrica e transitiva a vuoto, ma non sarebbe riflessiva

**Comment by CasuFrost** [comment]
![image](https://user-images.githubusercontent.com/97389762/216762116-7e9aecc0-c0ae-46e9-8437-d84769e06599.png)

l'affermazione evidenziata implica che ogni singolo elemento di A è in relazione con tutti gli altri elementi, è quindi ovvio che sia transitiva, se A=(1,2,3) esiste 1,2  esiste 2,3 ed esiste anche 1,3

**Comment by LuginiAndrea** [comment]
> ![image](https://user-images.githubusercontent.com/97389762/216762116-7e9aecc0-c0ae-46e9-8437-d84769e06599.png)
> 
> l'affermazione evidenziata implica che ogni singolo elemento di A è in relazione con tutti gli altri elementi, è quindi ovvio che sia transitiva, se A=(1,2,3) esiste 1,2 esiste 2,3 ed esiste anche 1,3

No attenzione: 
Ipotizza $$R = \left(1,2\right), \left(2,3\right), \left(3,2\right)$$ e controlliamo se rispettiamo le premesse:

- $1 \rightarrow \left(1,2\right)$: premessa rispettata
- $2 \rightarrow \left(2,3\right)$: premessa rispettata
- $3 \rightarrow \left(3,2\right)$: premessa rispettata
La relazione non è però transitiva in quanto manca $\left(1,3\right)$

L'affermazione evidenziata indica che per ogni elemento di A esiste _almeno_ un elemento tale che $aRb$, ma non che per ogni $b \in A, aRb$

**Comment by CasuFrost** [comment]
@LuginiAndrea ah ok hai ragione, grazie del chiarimento
